src/Language/PureScript/Ide/Imports.hs

```diff
@@ -49,8 +49,7 @@ import           Language.PureScript.Ide.Types
 import           Language.PureScript.Ide.Util
 import           System.IO.UTF8                     (writeUTF8FileT)
 
-data Import = Import P.ModuleName P.ImportDeclarationType (Maybe P.ModuleName)
-              deriving (Eq, Show)
```

Comment:
Could you clarify why this was moved to the `Types` module?

Reply:
To avoid a circular reference, `Ide.Imports` -> `Ide.Filter` `Ide.Filter.Imports` -> `Ide.Imports`

Reply:
Actually moved it back with further changes to take imports in a different form